charger/easee.go

```diff
@@ -72,10 +73,11 @@ func init() {
 // NewEaseeFromConfig creates a go-e charger from generic config
 func NewEaseeFromConfig(other map[string]interface{}) (api.Charger, error) {
 	cc := struct {
-		User     string
-		Password string
-		Charger  string
-		Timeout  time.Duration
+		User      string
+		Password  string
+		Charger   string
+		Timeout   time.Duration
+		Authorize bool
```

Comment:
Meaning default is current mode of operation, i.e. this is opt-in?

Reply:
Yes, this is opt-in only.